runtime/local/src/lib.rs

```diff
@@ -749,7 +752,7 @@ impl pallet_contracts::Config for Runtime {
     type CallStack = [pallet_contracts::Frame<Self>; 31];
     type WeightPrice = pallet_transaction_payment::Pallet<Self>;
     type WeightInfo = pallet_contracts::weights::SubstrateWeight<Self>;
-    type ChainExtension = DappsStakingChainExtension;
+    type ChainExtension = (DappsStakingExtension<Self>,);
```

Comment:
Why do you need <Self> here ?

Reply:
Syntactically it’s not required, but idiomatically it helps reader to understand that tuple expected.